client/package.json

```diff
@@ -60,14 +61,15 @@
   },
   "scripts": {
     "lingui": "lingui",
+    "prettier:check": "cd .. && prettier \"**/*.{ts,tsx,js,scss}\" --check && cd client",
     "contentful": "node pull-contentful-data.js",
```

Comment:
This is not the most DRY solution (as it literally just copies the prettier:check command from the root directory) but given that we are working within a package.json file, I couldn't think of any way to optimize this. Any thoughts?

Reply:
yeah i think it is fine! i blame having two package dot json's which is not your fault!

Reply:
oh wait i do have one idea, what if you actually changed it to `cd .. && yarn prettier:check && cd client`?

Reply:
I tried that and it didn't recognize `yarn prettier:check` even though it's defined in the parent directory's package.json... it seems like some linting mechanism only allows for references to functions that are defined within the same file?

Reply:
Oh wait! I just tried it again and it works :). Maybe I was running `cd .. && prettier:check && cd client` before, without the `yarn` which would make sense why it wouldn't be recognized.